buildpipeline/Core-Setup-Publish.json

```diff
@@ -314,10 +429,23 @@
     },
     "TeamName": {
       "value": "DotNetCore"
+    },
+    "VsoToolsRepo": {
```

Comment:
In the build definition, you set `PB_VsoToolsDir`, and then any repo step which uses that variable should be explicitly passed `/p:VsoToolsdir=$(PB_VsoToolsDir)`

Side note:  Is it possible to make these into packages which are consumed in the build rather than having to clone a separate repo?